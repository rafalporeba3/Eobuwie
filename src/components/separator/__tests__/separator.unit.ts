import { shallowMount, Wrapper } from '@vue/test-utils';
import cSeparator from '../separator.component.vue';

describe('Separator component', () => {
  let wrapper!: Wrapper<any>;

  beforeEach(() => {
    wrapper = shallowMount(cSeparator);
  });

  it('should render component with correct class', () => {
    expect(wrapper.classes()).toContain('c-separator');
  });

  it('should match snapshot', () => {
    expect(wrapper.html()).toMatchSnapshot();
  });
});
