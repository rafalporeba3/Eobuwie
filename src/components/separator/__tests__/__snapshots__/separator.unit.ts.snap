// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Separator component should match snapshot 1`] = `<div class="c-separator"></div>`;
